### PR TITLE
Add install progress

### PR DESCRIPTION
**Pull request by sickboy**
Experimental at best.

### CODE REVIEW

File: src/Update/Program.cs [hunk 1]
```diff
@@ -181,11 +184,25 @@ int main(string[] args)
                 this.ErrorIfThrows(() => File.Copy(Assembly.GetExecutingAssembly().Location, updateTarget, true),
                     "Failed to copy Update.exe to " + updateTarget);
 
-                await mgr.FullInstall(silentInstall);
+                var app = new Application();
```
Comment (paulcbetts) [inline src/Update/Program.cs]: Why are we creating a dummy window? We've already got a Window in AnimatedGifWindow

**Comment by sickboy** [comment]
Using the existing window now, probably could refactor a little more to move the app/window logic out of the Install method.

**Comment by sickboy** [comment]
Decoupled and cleaned up now. I miss RX ;)

**Comment by paulcbetts** [comment]
Into it. Thanks @sickboy!@

**Comment by sickboy** [comment]
Awesome, thanks @paulcbetts!

**Pull request merged**